app/design/frontend/Diana/default-theme/Magento_Cms/templates/store-selector-main.phtml

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/** @var \Magento\Framework\View\Element\Template $block */
+?>
+<div data-bind="scope: 'renderStores'" class="render-stores-wrapper">
+    <h2 class="title-store-selector">
+        <?= $block->escapeHtml(__('Sales are available at stores in ')) ?>
+        <span data-bind="text: selectedCity"></span>
+    </h2>
+    <div class="grid-of-stores" data-bind="foreach: paginatedElements">
+        <div class="available-store">
+            <a data-bind="attr: {href: link}">
+                <img class="image-store" data-bind="attr: {alt: name}" width="180px"
+                     src="<?= $block->getViewFileUrl('images/example-store.jpeg'); ?>" />
```

Comment:
@Dianaeva Please, change `<?= $block->getViewFileUrl('images/example-store.jpeg'); ?>` to Knockout way of receiving img src